src/main/java/mate/academy/model/MovieSession.java

```diff
@@ -0,0 +1,66 @@
+package mate.academy.model;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.FetchType;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.OneToOne;
+import jakarta.persistence.Table;
+import java.time.LocalDateTime;
+
+@Entity
+@Table(name = "movie_sessions")
+public class MovieSession {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @OneToOne (fetch = FetchType.LAZY)
+    private Movie movie;
+    @OneToOne (fetch = FetchType.LAZY)
+    private CinemaHall cinemaHall;
```

Comment:
Wrong relations between tables
```suggestion
    @OneToOne (fetch = FetchType.LAZY)
    private Movie movie;
    @OneToOne (fetch = FetchType.LAZY)
    private CinemaHall cinemaHall;
```